plugin-server/src/worker/ingestion/persons/person-update.ts

```diff
@@ -19,6 +19,10 @@ export interface PropertyUpdates {
 const NO_PERSON_UPDATE_EVENTS = new Set(['$exception', '$$heatmap'])
 const PERSON_EVENTS = new Set(['$identify', '$create_alias', '$merge_dangerously', '$set'])
 
+// GeoIP properties that should still trigger person updates even when other geoip properties are blocked
+// These are commonly used for segmentation and are worth keeping up-to-date
+const ALLOWED_GEOIP_PROPERTIES = new Set(['$geoip_country_name', '$geoip_city_name'])
```

Comment:
question: Should we start with just country name, as it's more important? Adding the city name would increase the update rate by 50-60% (vs 25-30% for just the country name).